FantasmoSDK/Classes/Extensions/ARFrame+Extension.swift

```diff
@@ -8,7 +8,10 @@
 import ARKit
 
 
-public extension ARFrame {
+extension ARFrame: FMFrame {
+    public var fmCamera: FMCamera {
```

Comment:
Do we need a different variable name here? Can this just be `camera`?

Reply:
Unfortunately not, because the extension from another protocol inside an extension doesn't work. If I called the variable `camera`, then the `var camera: ARCamera` inside an `ARFrame` wouldn't be recognized by the compiler as an `FMCamera` because this is done in an extension. Peeves me too, wish I had found a better way, but this is how I've managed it.